pkgs/by-name/as/asset-ripper/package.nix

```
@@ -0,0 +1,65 @@
+{
+  autoPatchelfHook,
+  buildDotnetModule,
+  dbus,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  lib,
+}:
+
+buildDotnetModule (finalAttrs: {
+  pname = "AssetRipper";
+  version = "1.1.3";
```

AssetRipper has new releases:
```suggestion
  version = "1.1.7";
```

---

Need to bump to .NET 9 for this by the way.

---

I tried updating, but `fetch-deps` wasn't working, and in fact it's not working for 1.1.3 i.e. at cdfc9705b45c8d707d1b39b52df04dee1779ca0b without changes. How did you generate `deps.nix`?

---

Good question. The passthru fetch-deps fails for me too. Looking through my shell history, I dropped into a dotnet sdk nix-shell within the project's source and ran approximately the following:

```shell
dotnet restore --packages out
nuget-to-nix out > deps.nix
```

---

Note that in comments. Failing to fetch deps automatically would mean that it cannot be updated properly with an update script, and people have to manually handle that, and they need to know how to do that manually by reading the relevant comments.

I am suspecting this be some bug of nuget-to-nix, but considering that currently there is work going on to retire nuget-to-nix (#362278), it may not worth digging into it. However, it would still be good if you can look into this.

---

I got 1.1.7 building. https://github.com/YoshiRulz/nixpkgs/commit/8ab183d027e177e53f233f673dc4797a443fe28f